fields/types/localfiles/LocalFilesType.js

```diff
@@ -170,7 +171,7 @@ localfiles.prototype.addToSchema = function() {
 		delete: function(element_id) {
 			if (exists(this, element_id)) {
 				var values = this.get(field.path);
-				var value = values.find(function (val) { return val._id === element_id; });
+				var value = values.find(function (val) { return val._id == element_id; });
```

Comment:
I do not see the goal of a comment because this is how javascript is suppose to work with == and ===.
Maybe a global note to remember how works mongoose and how to compare object ids